rules/Php84/Rector/Param/ExplicitNullableParamTypeRector.php

```diff
@@ -75,6 +75,6 @@ public function refactor(Node $node): ?Param
 
     public function provideMinPhpVersion(): int
     {
-        return PhpVersionFeature::DEPRECATE_IMPLICIT_NULLABLE_PARAM_TYPE;
```

Comment:
This constant become unused, but is part of the API, I don't know if you want to keep it.